CMakeLists.txt

```diff
@@ -206,7 +206,7 @@ if(VELOX_ENABLE_ABFS)
   if(AZURESDK_ROOT_DIR)
     list(APPEND CMAKE_PREFIX_PATH ${AZURESDK_ROOT_DIR})
   endif()
-  find_package(azure-storage-blobs-cpp CONFIG REQUIRED)
+  find_package(azure-storage-files-datalake-cpp CONFIG REQUIRED)
```

Comment:
nit: should it be versioned ?

Reply:
currently i don't see versioned is needed here when tried 12.6+ due to no broken chagne for used APIs

Reply:
Don't we need the `azure-storage-blobs-cpp` package?

Reply:
I am guessing `azure-storage-files-datalake-cpp` package includes both  libraries.